kubernetes/zenko/values.yaml

```diff
@@ -33,6 +33,12 @@ global:
   locationConstraints: {}
   replicationEndpoints: []
 
+s3Utils:
+  image:
+    repository: zenko/s3utils
+    tag: v0.8
```

Comment:
This tag will need to be bumped.

Reply:
it looks like there is already a `v0.9` from a couple months ago so likely `v0.10` unless you wanna go for the major version.

Reply:
That's true.